### PR TITLE
aws - elasticache replication group - tagging support

### PR DESCRIPTION
**Pull request by seavargas**
ElastiCache Replication Groups now supports the following actions and filters because universal tagging (AWS resource API wrapper) has been added 
- action: tag
- action: un-tag
- action: mark-for-op
- filter: marked-for-op

All replication group test cases have been re-recorded because the universal augmentation required for universal tagging adds an API call to describe tags when the `ElastiCacheReplicationGroup` is instantiated 

This feature was made in response to [issue 6834](https://github.com/cloud-custodian/cloud-custodian/issues/6834)

### CODE REVIEW

File: c7n/tags.py
Comment (JohnHillegass) [inline c7n/tags.py:8]: small typo here s/suport/support

**Comment by JohnHillegass** [comment]
Looks fine to me! Please let us know once you have accepted the CLA and it is approved

https://api.easycla.lfx.linuxfoundation.org/v2/repository-provider/github/sign/14202841/52837350/6858/#/?version=2

**Comment by seavargas** [comment]
thanks @JohnHillegass! CLA is currently pending Capital One sign off.

File: c7n/tags.py
Comment (seavargas) [reply c7n/tags.py]: thanks for catching this, new commit below adds a "p"

**Comment by kapilt** [comment]
> thanks @JohnHillegass! CLA is currently pending Capital One sign off.

fwiw, capital one is not involved in our cla process. its via the cloud native compute foundation where this project lives re current cla process.

**Review by kapilt — APPROVED** [review]
lgtm, ping if there's any issues re cla process.

**Comment by seavargas** [comment]
@kapilt roger that. I just assumed the pending sign off from a "CLA manager for my organization" as part of the Corporate Contributor process was from Capital One. thanks.

**Comment by kapilt** [comment]
@seavargas i'm not seeing any signatures you've done in the linux foundation's easy cla systems. you should be able to sign the individual license contributor agreement yourself. wrt to corporate setup, it requires more steps that would need to be completed, and its best to engage either with your opensource office if your org has one or directly with the easy cla support staff. if you need help with this let me know.

**Comment by seavargas** [comment]
@kapilt our internal folks say they've just added my github username to the corporate CLA tool

**Comment by seavargas** [comment]
@kapilt it appears the CLA is now passing 🙌

**Comment by kapilt** [comment]
@JohnHillegass thanks!

**Pull request merged**
